### ISSUE TITLE
Confusion about "Account" options.

### ISSUE THREAD

**Issue by s3w47m88** [firefly-iii/firefly-iii#2275]
I am running Firefly III version 4.7.5

**Description**
Under Accounts it gives Asset, Expense, Revenue and Liability. But our Checking account is used for Expenses and Revenue deposits. I'm confused as to why you separate them and what I'm suppose to use...

LOVE this application. Will be contributing some features my team and I have built but are testing including automatic importing from PayPal!

**Comment by JC5** [comment]
Think about it like bank account statements. You're checking account lists all transactions going to and coming from (resp) expense and revenue accounts. But these revenue/expense accounts have their own overviews as well.

<img width="1015" alt="Screenshot 2019-05-18 at 05 56 41" src="https://user-images.githubusercontent.com/5889984/57964206-bba4aa00-7931-11e9-8274-39ab04f507e8.png">

This is a pretty bland example of a list of expenses on your checking account.

<img width="1197" alt="Screenshot 2019-05-18 at 05 57 31" src="https://user-images.githubusercontent.com/5889984/57964219-dd059600-7931-11e9-9f2d-455b13e1b075.png">

The cafe central account lists all of the expenses made there.

Software like GNUCash et al use a slightly different definition for accounts I think, but they're boring.

**Comment by JC5** [comment]
Thanks for the compliments! They're always appreciated :).

If you're planning to contribute something import related, drop me a line first. I'm changing the way imports work in future releases and it would be a shame if you did a lot of work for nothing.

https://github.com/firefly-iii/firefly-iii/projects/5#card-19937520

**Comment by s3w47m88** [comment]
Okay so Expense Accounts are vendors (people or companies) I pay?

Then would a Checking account be a Asset Account in Firefly?

Thanks!

Okay I'll be in touch. We already built it out, I'll give you a demo in the near future. Automatically connects to PayPal but currently only one account at a time - working on multiple accounts. I built https://fixpaypal.com and then realized it would be a good package for Laravel and especially Firefly IMO. Happy to contribute it as Open Source when you've had a chance to review it.
